docs/topics/js/js-project-setup.md

```diff
@@ -550,6 +550,82 @@ rootProject.plugins.withType(org.jetbrains.kotlin.gradle.targets.js.yarn.YarnPlu
 </tab>
 </tabs>
 
+### Version locking via kotlin-js-store
+
+> Version locking via `kotlin-js-store` is available since Kotlin 1.6.10.
+>
+{type="note"}
+
+The `kotlin-js-store` directory in a Kotlin/JS project root is necessary for version locking because it holds the 
+`yarn.lock` file inside. To follow a [recommended practice](https://classic.yarnpkg.com/blog/2016/11/24/lockfiles-for-all/), 
+commit `kotlin-js-store` and its contents to your version control system. It ensures that your application is being 
+built with the exact same dependency tree on all machines.
+
+If needed, you can change both directory and lockfile names in the build script:
```

Comment:
@ilgonmic Could you please also suggest a name of a gradle task that regenerates `kotlin-js-store`? Our colleague asked to provide this information.